README.md

```diff
@@ -26,6 +26,34 @@ The code base is still pure Python, which means it will still run in CPython and
 At the Wiki page, you will also find out how to interface with PyBoy from your own project: [Wiki](https://github.com/Baekalfen/PyBoy/wiki).
 
 
+
+# OpenAI Interface
```

Comment:
This should be moved to the Wifi when we merge, but for now, we can leave it here.